banners/neue_wege_2024/Banner.html

```diff
@@ -289,85 +336,69 @@
 		}
 	}
 
-	// returns 0 if no user group, 1 for group 1, 2 for group 2
-	const get_usergroup = () => {
-		// user group 1: user is active or passive reviewer
-		// user group 2: not a reviewer but at least 1 edit
-		
-		var userGroups = mw.config.get('wgUserGroups');
+	/**
+	 * USER_GROUP_REVIEWER: user is active or passive reviewer
+	 * USER_GROUP_HAS_EDITS: not a reviewer but at least 1 edit
+	 * DO_NOT_SHOW_BANNER: not one of the targeted groups
+	 *
+	 * @returns {int}
+	 */
+	const getUserGroup = () => {
+		const urlParams = new URLSearchParams( window.location.search );
 
-		// Check if userGroups exists and includes either "editor" or "autoreview"
-		if (userGroups && (userGroups.includes('editor') || userGroups.includes('autoreview'))) {
-			return "2";
+		if ( urlParams.has( 'group' ) ) {
+			return parseInt( urlParams.get( 'group' ) );
+		}
+
+		const userGroups = mw.config.get( 'wgUserGroups' );
+
+		if ( userGroups && ( userGroups.includes( 'editor' ) || userGroups.includes( 'autoreview' ) ) ) {
```

Comment:
from the PM in the chat on Monday evening:
```
user group 1: at least 1 edit (group doesn't matter)
user group 2: editor && autoreview
```
```suggestion
		if ( userGroups && ( userGroups.includes( 'editor' ) && userGroups.includes( 'autoreview' ) ) ) {
```
+ adapting the comments saying "or"